tests/test_patchqueue.py

```diff
@@ -50,13 +51,103 @@ def test_rewrite_spec(self):
         self.assertIn("Patch2: third.patch\n", rewritten)
 
     def test_autosetup_present(self):
+        """Patchqueue application succeeds if %autosetup is present"""
         spec = Spec("tests/data/manifest/branding-xenserver.spec",
                     check_package_name=False)
         patches = ["first.patch"]
-        planex.patchqueue.expand_patchqueue(spec, patches)
```

Comment:
Yes, this test was verifying that expand_patchqueue() did not raise an exception.   However pylint complained that the test did not use self and therefore didn't need to be a member.   This change seemed better than just saying 'self.assertTrue(True)' to shut pylint up...